quest.karaganov/src/main/java/ru/jru/quest/karaganov/filter/StartFilter.java

```diff
@@ -0,0 +1,49 @@
+package ru.jru.quest.karaganov.filter;
+
+import lombok.Data;
+
+import javax.servlet.FilterChain;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.*;
+import java.io.IOException;
+import java.util.Enumeration;
+
+@Data
+@WebFilter(urlPatterns = "/gameServlet")
+public class StartFilter extends HttpFilter {
+
+    @Override
+    public void init() throws ServletException {
```

Comment:
It seems that `init` method has a default implementation in Filter interface 
https://javaee.github.io/javaee-spec/javadocs/javax/servlet/Filter.html